test/integration/agent/route_test.go

```diff
@@ -388,25 +335,19 @@ func TestRouteTablePolicyOnly(t *testing.T) {
 		t.Error(err)
 	}
 	//verify gw IP
-	gwName := nodeConfig.GatewayConfig.Name
-	gwIPOut, err := ExecOutputTrim(fmt.Sprintf("ip addr show %s", gwName))
-	if err != nil {
-		t.Error(err)
-	}
-	gwIP := net.IPNet{
-		IP:   nodeConfig.NodeIPAddr.IP,
-		Mask: net.CIDRMask(32, 32),
-	}
-	assert.Contains(t, gwIPOut, gwIP.String())
-	// verify default routes and neigh
-	expRoute := strings.Join(strings.Fields(
-		"default via 169.254.253.1 dev antrea-gw0 onlink"), "")
-	routeOut, err := ExecOutputTrim(fmt.Sprintf("ip route show table %d", svcTblIdx))
-	assert.Equal(t, expRoute, routeOut)
-	expNeigh := strings.Join(strings.Fields(
-		"169.254.253.1 dev antrea-gw0 lladdr 12:34:56:78:9a:bc PERMANENT"), "")
-	neighOut, err := ExecOutputTrim(fmt.Sprintf("ip neigh | grep %s", gwName))
-	assert.Equal(t, expNeigh, neighOut)
+	// TODO: no need ??
```

Comment:
what information do we need to determine it?

Reply:
Test condition is disabled by mistake. NodeIP will be needed still to satisfy spoofGuard.